inc/dbmysqliterator.class.php

```diff
@@ -581,12 +581,12 @@ private function analyzeCriterionValue($value) {
     *
     * @since 9.4.0
     *
-    * @param array $joinarray Array of joins to analyze
+    * @param array $joinarray Array of joins to analyse
     *       [jointype => [table => criteria]]
     *
     * @return string
     */
-   public function analyzeJoins(array $joinarray) {
+   public function analyseJoins(array $joinarray) {
```

Comment:
As this function is public, should we handle a deprecation state ? (I do not see usages on plugins I have on my instance for information)

Reply:
I do not think, it does not really make  sense to override this